portfolio/src/main/webapp/script.js

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Week 1 **/
+/** Week 1 */
```

Comment:
I think comments should be /** and **/ ? I'm unsure

Reply:
Nevermind comments are /* and */

Reply:
Yep, actually the number of stars determines the kind of comment.

/** */ comments are Javadoc style comments and are often used to generate documentation for your API
/* */ are just block comments which are not necessarily documentation